00 - Contribute to the Obsidian Hub/Obsidian Settings for Contributors.md

```diff
@@ -0,0 +1,17 @@
+# Where is the .obsidian directory?
+
+The `.obsidian` folder is ignored by git, so anyone loading this repo in to Obsidian, in order to modify content, needs to set up the Obsidian settings themselves.
+
+## Obsidian Settings for Editing Content
+
+For editing content, the requirements are minimal.
+
+They are documented in the [CONTRIBUTING](https://publish.obsidian.md/hub/CONTRIBUTING) page, under section "Setup & Vault Consistency".
+
+## Argentum's Settings
+
+Argentum has kindly provided a Zipped copy of the `.obsidian` directory from a properly-configured hub vault.
```

Comment:
I think this can be removed because of line 15.

Reply:
Fixed in a1be56190c8caa6635bb70836d157db79f57a106